test/functional/apps/discover/index.ts

```diff
@@ -23,38 +23,41 @@ export default function ({ getService, loadTestFile }: FtrProviderContext) {
       await esArchiver.unload('test/functional/fixtures/es_archiver/logstash_functional');
     });
 
-    loadTestFile(require.resolve('./_saved_queries'));
-    loadTestFile(require.resolve('./_discover'));
-    loadTestFile(require.resolve('./_discover_histogram'));
-    loadTestFile(require.resolve('./_doc_table'));
-    loadTestFile(require.resolve('./_doc_table_newline'));
-    loadTestFile(require.resolve('./_filter_editor'));
-    loadTestFile(require.resolve('./_errors'));
-    loadTestFile(require.resolve('./_field_data'));
-    loadTestFile(require.resolve('./_field_data_with_fields_api'));
-    loadTestFile(require.resolve('./_shared_links'));
-    loadTestFile(require.resolve('./_sidebar'));
-    loadTestFile(require.resolve('./_source_filters'));
-    loadTestFile(require.resolve('./_large_string'));
-    loadTestFile(require.resolve('./_inspector'));
-    loadTestFile(require.resolve('./_classic_table_doc_navigation'));
-    loadTestFile(require.resolve('./_date_nanos'));
-    loadTestFile(require.resolve('./_date_nanos_mixed'));
-    loadTestFile(require.resolve('./_indexpattern_without_timefield'));
-    loadTestFile(require.resolve('./_discover_fields_api'));
-    loadTestFile(require.resolve('./_data_grid'));
-    loadTestFile(require.resolve('./_data_grid_context'));
-    loadTestFile(require.resolve('./_data_grid_field_data'));
-    loadTestFile(require.resolve('./_data_grid_doc_navigation'));
-    loadTestFile(require.resolve('./_data_grid_doc_table'));
-    loadTestFile(require.resolve('./_indexpattern_with_unmapped_fields'));
-    loadTestFile(require.resolve('./_runtime_fields_editor'));
-    loadTestFile(require.resolve('./_huge_fields'));
-    loadTestFile(require.resolve('./_date_nested'));
-    loadTestFile(require.resolve('./_search_on_page_load'));
-    loadTestFile(require.resolve('./_chart_hidden'));
-    loadTestFile(require.resolve('./_context_encoded_url_param'));
-    loadTestFile(require.resolve('./_data_view_editor'));
-    loadTestFile(require.resolve('./_empty_state'));
+    if (config.get('esTestCluster.ccs')) {
+      loadTestFile(require.resolve('./_data_view_editor'));
+    } else {
```

Comment:
Is this check necessary? because I think you've prepared _data_view_editor .ts to work with CCS and non-CCS configuration, but that if check would only run it with CCS config?

Reply:
The check is necessary. The test is run in both configs at line 61, this same file is run if the config is not CCS. When CCS is enabled, we only want the CCS tests to run. I did, however, miss the `saved_queries` test so I will add that to work when ccs is enabled and push.